src/components/clusters/Clusters.tsx

```diff
@@ -115,12 +126,7 @@ const Clusters: React.FC<ClustersProps> = ({ history }) => {
         </>
       );
   }
-  return (
-    <>
-      {body}
-      {isModalOpen && <NewClusterModal closeModal={closeModal} history={history} />}
-    </>
-  );
+  return <>{body}</>;
```

Comment:
```suggestion
  return body;
```

or we can just return directly in switch/case statements

Reply:
Done